state/package_test.go

```diff
@@ -20,3 +33,14 @@ func TestPackage(t *testing.T) {
 	}
 	coretesting.MgoTestPackage(t)
 }
+
+func SetModelTypeToCAAS(c *gc.C, st *State, m *Model) {
```

Comment:
I wanted to do this is a suite, but there is an import cycle state -> state/testing -> state if you use the `StateTest` base.